src/index.css

```diff
@@ -18,4 +18,5 @@ body,
   height: 100%;
   width: 100%;
   margin: 0;
+  overflow: hidden;
```

Comment:
Please find out which element causes the overflow and fix them instead, to avoid clipping them.